src/lib/localization.js

```diff
@@ -0,0 +1,52 @@
+import { L10nError } from './errors';
+
+export class Localization {
```

Comment:
I'm not sure what's the value of a class that has only private methods and doesn't give anything to `HTMLLocalization`. Maybe it should be just a bunch of functions?